modules/zip_importer/includes/importer.inc

```diff
@@ -178,10 +178,29 @@ class ZipBatchImporter extends IslandoraBatchImporter {
 class ZipBatchImportObject extends IslandoraImportObject {
 
   protected $mods;
-  protected static $MARC2MODS;
```

Comment:
No worries. I may be overthinking, but just thinking about other issues which may pop up related to this. I'm not aware of any implementation right off, but there could exist an implementation subclassing this one which should be similarly updated... Given they would've likely been setting the paths in the constructor, it's not clear what one should do to update... Implement `initializePaths()` with their overrides, implement "__wakeup" to parallel what they're doing in their constructor? Something else?

Yeah, I'm probably overthinking this.